src/content/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations.mdx

```diff
@@ -40,83 +39,74 @@ import osUbuntu from 'images/os_icon_ubuntu.webp'
+* [Kafka](/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration)
+* [Memcached](/docs/integrations/host-integrations/host-integrations-list/memcached-monitoring-integration)
+* [MongoDB](/docs/integrations/host-integrations/host-integrations-list/mongodb-monitoring-integration)
+* [MySQL](/docs/integrations/host-integrations/host-integrations-list/mysql-monitoring-integration)
+* [NGINX](/docs/integrations/host-integrations/host-integrations-list/nginx-monitoring-integration)
+* [PostgreSQL](/docs/integrations/host-integrations/host-integrations-list/postgresql-monitoring-integration)
+* [RabbitMQ](/docs/integrations/host-integrations/host-integrations-list/rabbitmq-monitoring-integration) 
+* [Redis](/docs/integrations/host-integrations/host-integrations-list/redis-monitoring-integration)
 
-All on-host integrations require that our infrastructure agent is [installed on the host](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent). Besides collecting system's data, the agent acts as a forwarder for integrations's data, and can forward [logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent).
+There are different ways to install these integrations, which will be explained below. For technical details about the file structure and configuration, see [Technical details](#technical-details).
 
-For example, for SQL Server:
-- Host 1 with the infrastructure agent installed needs an outbound rule for port `1433`.
-- Host 2 with SQL Server needs an inbound rule for port `1433`.
+## Find data [#find-data]
 
-In this example, host 1 pulls data from host 2.
+Data reported by these integrations can be found by going to: **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Third-party services**. 
 
-<img
-  title="Agents and New Relic"
-  alt="Agents and New Relic"
-  src={infrastructureAgentsandNr}
-/>
+## Installation methods [#install]
 
-<figcaption>
-  The infrastructure agent and its integrations collect data from the system and core services, and is able to [forward logs](/docs/logs/enable-log-monitoring-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent) to New Relic. Backend application metrics (APM) are collected by separate [APM agents](/docs/apm/new-relic-apm/getting-started/introduction-apm). Notice how each integration and forwarder feed [different data types](/docs/using-new-relic/data/understand-data/new-relic-data-types) in the New Relic database (NRDB).
-</figcaption>
+Your services may be running on a single physical host, on a VM, or in a container orchestrated by Kubernetes or Amazon ECS. Choose your desired installation method.
 
-## Quick start [#quick]
+After you do the basic install of an integration, you must return to the documentation for a specific integration to read the configuration instructions. 
```

Comment:
@zuluecho9 this is where you could link to a new section at the end that tells users to do the configurations (and then list all the software with links to the docs). OK, I now see that you have this link at the bottom of each collapser, but you could just use that to link out the the new section that lists all the integrations with links).